docker/test

```diff
@@ -75,29 +74,51 @@ then
   exit 1
 fi
 
+if [ "$COMPOSE_ARGS" == "-f $DIR/docker-compose-postgres-cs9.yml" ]; then
+  INIT_CONTAINER="1"
+fi
+
 # PROJ_NAME should be set in a jenkins environment. It allows multiple
 #  instances of the compose to run without clobbering each other.
 cd $DIR
-docker-compose $PROJ_NAME stop
-docker-compose $PROJ_NAME rm -f
 
-if [ "$USE_CACHE" != "1" ]; then
-  docker-compose $COMPOSE_ARGS pull
-fi
 
 SERVICE_PORTS_ARG=""
 if [ "$EXPOSE_PORTS" == "1" ]; then
   SERVICE_PORTS_ARG="--service-ports"
 fi
 
-if [ "$DETACHED" == "1" ]; then
+if [ "$INIT_CONTAINER" == "1" ]; then # For CS9
+  podman-compose $PROJ_NAME stop
+  podman-compose $PROJ_NAME rm -f
+
+  if [ "$USE_CACHE" != "1" ]; then
+    podman-compose $COMPOSE_ARGS pull
+  fi
+  podman-compose $PROJ_NAME $COMPOSE_ARGS up -d
+  sleep # Waiting to start systemd
```

Comment:
how does this work? doesn't this need an argument for duration? (e.g. 2 seconds)

Reply:
Also, `wait <job_id>` may be a better option than `sleep`, since it will wait exactly until `podman-compose up` is done, instead of having to guess how long it may take

Reply:
TL;DR: systemd takes some time to start :)

Few months ago, I updated the deployment of Candlepin in the 'services' image of the cockpit CI, and the deployment is done as container in a Fedora CoreOS distro. When you start the new container using (directly or indirectly) `podman run`, the `podman` command returns when the container is started; however, the entry point (systemd in this case) may take a while to start and be fully operational, so doing
```
$ podman run ...
$ podman exec systemctl
```
may fail because systemd is not ready yet. If the host that runs the container is not a baremetal system (e.g. a VM), the systemd startup may take longer. Hence, sleeping few seconds (I chose 5 for cockpit, and so far has been working fine) is "good enough" to ensure systemd is operational and responsive. Yes, it feels a bit of a hack, however not that bad of a hack.

`podman wait` is not actually useful here, since it waits for the container to stop, which in practice it means when systemd is stopped (and thus not what you want).

Reference: https://github.com/cockpit-project/bots/pull/3981